_includes/_global/required-roles.md

```diff
@@ -0,0 +1,25 @@
+**Granted at the Organization level**
+
+ `forseti_inventory` and `forseti_scanner` require:
+ 
+ * `roles/browser`
+ * `roles/compute.networkViewer`
+ * `roles/iam.securityReviewer`
+ * `roles/appengine.appViewer`
+ * `roles/servicemanagement.quotaViewer`
+ * `roles/cloudsql.viewer`
+ 
+ `forseti_enforcer` requires:
+ 
+ * `roles/compute.securityAdmin`
+
+**Granted on the project where Forseti Security is deployed**
+
+ `forseti_inventory` and `forseti_scanner` require:
```

Comment:
enforcer will also require these for reading firewall policies from GCS and writing rollback snapshot policies to GCS (soon).